app/faucet/templates/shared/faucet_unlock_metamask.html

```diff
@@ -0,0 +1,35 @@
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
```

Comment:
any way we can just use the current `unlock_metamask.html` shared module?

Reply:
it would require a conditional so the header could be swapped out depending upon the use-case... which is why I opted for a new view
<h3>Apply for a faucet distribution</h3>